src/merkle/path_set.rs

```diff
@@ -150,7 +151,7 @@ impl MerklePathSet {
     pub fn add_path(
         &mut self,
         index_value: u64,
-        value: Word,
+        value: RpoDigest,
```

Comment:
If we are treating leaves as `Word`'s I think we should do it consistently and have `value` be a `Word` here as well.

This would apply to `update_leaf()` method too.